app-text/diffpdf/files/diffpdf-2.1.3-qt6.patch

```diff
@@ -0,0 +1,407 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+new file mode 100644
+index 0000000..c7e6d76
+--- /dev/null
++++ b/CMakeLists.txt
+@@ -0,0 +1,25 @@
++cmake_minimum_required(VERSION 3.28)
```

Comment:
I guess 3.20 would be completely fine and may enable other distros to easily consider your patch as well.

Reply:
Changed to 3.16. Tested to compile against that on Debian 12, older versions didn't work due to some dependency requiring 3.16 or later.